sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeHealthcareAsyncClient.java

```diff
@@ -74,12 +75,17 @@
         try {
             inputDocumentsValidation(documents);
             String modelVersion = null;
+            Duration pollInterval = DEFAULT_POLL_INTERVAL;
             if (options != null) {
                 modelVersion = options.getModelVersion();
+                pollInterval = options.getPollInterval();
```

Comment:
by default, options.getPollInterval() ==> DEFAULT_POLL_INTERVAL, so if options is null, we use the defaultValue, if options is not null we use custom polling interval if user has set it, otherwise, use the DEFAULT_POLL_INTERVAL.

Reply:
It is already there: https://github.com/Azure/azure-sdk-for-java/pull/17267/files#diff-659663d93baf5cac89b0841764fb0a0999f5461346bce5cc25fa1f8e4e0a2137R136

This check has a different purpose.  If 'options' is null, we are using DEFAULT_POLL_INTERVAL,  otherwise, we use what user defined custom poll interval. If user does not set any value, the pollInterval is still  DEFAULT_POLL_INTERVAL